include/dav1d/picture.rs

```diff
@@ -520,7 +519,7 @@ impl From<Rav1dPicture> for Dav1dPicture {
             seq_hdr_ref: seq_hdr.map(RawArc::from_arc),
             content_light_ref: content_light.map(RawArc::from_arc),
             mastering_display_ref: mastering_display.map(RawArc::from_arc),
-            itut_t35_ref: Some(itut_t35).map(RawArc::from_arc),
+            itut_t35_ref: Some(RawArc::from_arc(itut_t35)),
```

Comment:
All of the other fields here are `.map(RawArc::from_arc)`, so I'd prefer to keep that consistency.